src/components/form/FormFieldDescription.tsx

```diff
@@ -29,6 +29,9 @@ interface Props {
   validation?: `${FormFieldValidation}`;
 }
 
+// TODO do we need aria-live="polite" ?
+// TODO we probably need to add some ellipsis handling
```

Comment:
Do we want to truncate the description, or allow it to wrap multiple lines? I guess that is a question for Marcio.

Reply:
Actually you are right, we don't want to truncate, we want to wrap, we already aligned on that with Marcio the other day. So we need to handle wrapping because it doesn't currently, but maybe it's just a matter of me rebasing on your branch to make it work since you added the width to FormField.